### ISSUE TITLE
Platform comparsion fails on Windows/MINGW32

### ISSUE THREAD

**Issue by Ultrabenosaurus** [raylee/tldr#6]
While the script does work after changing the comparison operator, it seems that the bash-like shell installed with git does not include support for `=~` comparisons. When I first tried the script after installation I received the following error:

```
/c/Users/Dan/bin/tldr: line 159: conditional binary operator expected
/c/Users/Dan/bin/tldr: line 159: syntax error near `=~'
/c/Users/Dan/bin/tldr: line 159: `    if [[ $desc =~ \"$platform\" ]]; then'
```

After deleting the `~` it began showing content, but no matter which operator I try it always takes several seconds (even for pages already downloaded) and states the following before the page's content:

```
tldr page man not found in common or common, using page from platform common instead
```

As for specifics regarding my platform:
Windows 10 Home 64-bit
`uname -s` > MINGW32_NT-6.2
`git --version` > 1.9.4.msysgit.0

**Comment by raylee** [comment]
Good effort! Though the =~ operator is being used for a regex comparison (but only for a substring) so changing it to a strict equality test won't help, as you found out. It's a feature available in recent bash versions, but apparently not in the one git bash for Windows you have. Regardless, the intent of those lines are to see if the local platform name is in a string that looks like (example): 
    `"name":"netstat","platform":["linux","osx"]},`
which comes from the index.json file.

I'll give a think on how I can test for what the script needs without using a substring match. The solution, though, will likely involve calling more external programs (such as cut and grep), which I suspect is the source of your second issue, the long time it takes to do anything.

If you have a chance, I'd appreciate knowing if this is still an issue on the latest git/bash bundle for Windows, which is up to 2.7.0 now: https://git-for-windows.github.io/

**Comment by Ultrabenosaurus** [comment]
Yeah, I discovered the purpose of it when I looked it up and found it was a fairly new thing. I didn't realise my git was so out of date...

I've installed 2.7.0 and it's... different. I like that I can make it transparent!

Anyway, I redownloaded `tldr` to get rid of whatever changes I'd made earlier and it's working perfectly, even the colours and other formatting are displaying now. Neither the regex operator nor the "not found" message are giving me issues. 

Now to customise this new prompt...

**Comment by Ultrabenosaurus** [comment]
(closed because it works if you actually keep git up-to-date)

**Comment by raylee** [comment]
Awesome, thanks for testing and the report!

**Comment by Ultrabenosaurus** [comment]
Different topic, but I just noticed the `-p` flag isn't mentioned in the help text of the actual program.

**Comment by raylee** [comment]
Good catch, I've added that to my list. I'll be sitting down and giving tldr some attention this weekend and will add the text for it then. Thanks!
